content/tracing/setup/javascript.md

```diff
@@ -0,0 +1,164 @@
+aliases:
+- /tracing/javascript/
+- /tracing/languages/javascript/
+further_reading:
+- link: "https://github.com/DataDog/dd-trace-js"
+  tag: "Github"
+  text: Source code
+- link: "tracing/visualization/"
+  tag: "Use the APM UI"
+  text: "Explore your services, resources and traces"
+---
+
+<div class="alert alert-warning">
+JavaScript APM is currently in <strong>beta</strong>.
+</div>
+
+## Getting started
+
+For descriptions of terminology used in APM, take a look at the [official documentation][visualization docs].
+
+For details about contributing, check out the [development guide][development docs].
+
+[visualization docs]: https://docs.datadoghq.com/tracing/visualization/
+[development docs]: https://github.com/DataDog/dd-trace-js/blob/master/README.md#development
+
+### Installation
+
+To begin tracing Node.js applications, first [install and configure the Datadog Agent](/tracing/setup/) (see additional documentation for [tracing Docker applications](/tracing/setup/docker/)).
+
+Next, install the Datadog Tracing library using npm:
+
+```sh
+npm install --save dd-trace
+```
+
+Finally, import the tracer and instrument your code!
+
+### Example
+
+```js
+const tracer = require('dd-trace').init()
+
+tracer
+  .trace('web.request', {
+    service: 'my_service'
+  })
+  .then(span => {
+    span.setTag('my_tag', 'my_value')
+    span.finish()
+  })
+```
+
+### Configuration Options
+
+Options can be configured as a parameter to the `init()` method or as environment variables.
+
+| Config        | Environment Variable         | Default   | Description |
+| ------------- | ---------------------------- | --------- | ----------- |
+| debug         | DD_TRACE_DEBUG               | false     | Enable debug logging in the tracer. |
+| service       | DD_SERVICE_NAME              |           | The service name to be used for this program. |
+| hostname      | DD_TRACE_AGENT_HOSTNAME      | localhost | The address of the trace agent that the tracer will submit to. |
+| port          | DD_TRACE_AGENT_PORT          | 8126      | The port of the trace agent that the tracer will submit to. |
+| flushInterval |                              | 2000      | Interval in milliseconds at which the tracer will submit traces to the agent. |
+| experimental  |                              | {}        | Experimental features can be enabled all at once using boolean `true` or individually using key/value pairs. Available experimental features: `asyncHooks`. |
+| plugins       |                              | true      | Whether or not to enable automatic instrumentation of external libraries using the built-in plugins. |
+
+## OpenTracing
+
+This library is OpenTracing compliant. Once the tracer is initialized
+it can be used as a global tracer.
+
+```js
+const tracer = require('dd-trace').init()
+const opentracing = require('opentracing')
+
+opentracing.initGlobalTracer(tracer)
+```
+
+The tracer will then be available with `opentracing.globalTracer()`.
```

Comment:
Here we should make a call, or we suggest to use our API or OpenTracing API. The options we have are:
1) we remove this line and we're good. No need to explain the OpenTracing API but only the way how we have to initialize our tracer
2) we write a full tracing example using the OpenTracing API. In that case, we should remove the initial Manual Instrumentation example because misleading

As a context, our API provides more methods and in-process context propagation, allowing easier instrumentation. In this step I would advocate for (1), because our API is a bit more easy to use. Anyway, developers using OpenTracing API, already have full compatibility from our Tracer.

@rochdev @bradmenezes @lejulie thoughts?